Algorithm/QCAlgorithm.cs

```diff
@@ -559,6 +559,18 @@ public virtual void PostInitialize()
                 Benchmark = BrokerageModel.GetBenchmark(Securities);
             }
 
+            // Check benchmark timezone against algorithm timezone to warn for misaligned statistics
+            if (Benchmark is SecurityBenchmark securityBenchmark)
+            {
+                // Only warn on daily resolutions as its statistics will suffer the most
+                var subscription = securityBenchmark.Security.Subscriptions.OrderByDescending(x => x.Resolution).FirstOrDefault();
+                if (subscription?.Resolution == Resolution.Daily && subscription.DataTimeZone != TimeZone)
```

Comment:
I think It's not the benchmark resolution we should check against but the users algorithms resolutions being used? The benchmark will be hourly or less in live trading. The `securityBenchmark.Security.Subscriptions` are empty at this stage btw, it's after selection that we create the subscriptions, in `UniverseSelection.cs`. We should still be able to get the time zone here asking the MHDB though. 
We should add a test covering this guy 👍